### ISSUE TITLE
Add IIFEs with Gulp

### ISSUE THREAD

**Issue by alisonmonteiro** [pipocadigital/grao-de-milho#21]
Add IIFEs automatically with Gulp using [gulp-iife](http://npm.im/gulp-iife)

Then, instead of doing this for each file:

```js
;(function() {
  console.log('something');
}());
```

We can do just:

```js
console.log('something');
```

**Comment by felipesousa** [comment]
@alisonmonteiro you resolve this implementation? I can build this feature. 🏌️

**Comment by alisonmonteiro** [comment]
@felipesousa No, I didn't. PRs are more than welcome. 😆
Thanks!

**Comment by nilkesede** [comment]
This was solved on 1dff6d9239f105f3a2f13305ea23acea9f564c13. Can this question be closed?

**Comment by alisonmonteiro** [comment]
[`1dff6d9`](https://github.com/pipocadigital/grao-de-milho/commit/1dff6d9239f105f3a2f13305ea23acea9f564c13) fixes this.
